webanno-ui-project/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/project/ProjectUsersPanel.java

```diff
@@ -409,6 +437,7 @@ public void onSubmit()
                 public void onSubmit()
                 {
                     UserDetailForm.this.setVisible(false);
+                    filterText.setModel(Model.of(""));
```

Comment:
Same as above - instead of setting a new model with an empty string, change the variable that the widget is bound to.